### PR TITLE
Bootstrap3 final reorg

### PR DESCRIPTION
**Pull request by biyeun**
last bit of refactoring and reorg before bootstrap3 migration work begins

currently testing on staging

@benrudolph

### CODE REVIEW

File: corehq/apps/hqwebapp/urls.py [hunk 1]
```diff
@@ -37,6 +37,6 @@
     url(r'^login/mobile/$', 'domain_login', name='domain_mobile_login', 
         kwargs={'template_name': 'login_and_password/mobile_login.html'}),
     url(r'^retreive_download/(?P<download_id>[0-9a-fA-Z]{25,32})/$', 
-        'retrieve_download', {'template': 'hqwebapp/file_download.html' },
+        'retrieve_download', {'template': 'style/includes/file_download.html' },
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/urls.py]: E202 whitespace before '}'

**Comment by benrudolph** [comment]
:+1: from me

**Comment by biyeun** [comment]
should be good to merge after build completes :+1:

**Comment by benrudolph** [comment]
:boom:

**Pull request merged**